magento/inventory
# MSI-877: Creating Source should not be allowed if Source Code is already used.

### Description
Currently it's not possible to create Source with non-unique source code already. This pull request just makes sources creating(updating) process more clear for admin user.

### Fixed Issues (if relevant)
1. magento-engcom/msi#877: Creating Source should not be allowed if Source Code is already used.

### Manual testing scenarios
1. Login to admin area.
2. Navigate to Store > Manage Sources
3. Add new source with code "test_source".
4. Save source.
5. Check, source successfully has been saved. "The Source has been saved." message appears.
6. Add another one source with the same code "test_source".
7. Save source.
8. Check, source hasn't been saved. "Source with code: "test_source" already exists." message appears.
9. Change source code to "test_source_1".
10. Save source.
11. Check, source successfully has been saved. "The Source has been saved." message appears.
